app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    cmd, source_file, target_file = command.split()
+    if cmd != "cp" or source_file == target_file:
+        return
+    try:
+        with (open(source_file, "r") as source,
+              open(target_file, "w") as target):
+            target.write(source.read())
+    except UnicodeError:
+        print("Error while copying file")
```

Comment:
While handling the `UnicodeError` is a good practice, this error message is not very informative. Consider including more details in the error message, such as the names of the source and target files, and the specific error message from the exception.